Split acdc dcdc into acdc and dcdc 2

Misca1234 opened the pull request:
Replacement of push 
https://github.com/KiCad/kicad-packages3D/pull/285

These models are updated after the correction in 
https://github.com/easyw/kicad-3d-models-in-freecad/pull/162



------------
Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items as you complete them:

- [ ] A screenshot of the model(s) aligned with KiCad footprints is *very useful*
- [ ] Was the 3D model(s) created in a mechanical modelling program?
- [ ] Source files (or scripts) have been contributed 
- [ ] Step model submitted in 1:1 scale
- [ ] WRL model submitted in 1:2.54 scale
- [ ] 3D model alignment checked against KiCad footprint
- [ ] WRL model uses correct material properties
- [ ] Geometry check performed against mechanical models

Shackmeister commented:
Hi @Misca1234 
The models with bend leads are not functioning properly
you can see this on
Converter_DCDC_RECOM_R-78HB-0.5L_THT
Converter_DCDC_RECOM_R5xxxDA_THT

following are missing models
Converter_DCDC_TRACO_TEN20-xxxx-N4_THT

following seems to not fit properly (might be a footprint problem)
Converter_ACDC_MeanWell_IRM-20-xx_THT

![converter_acdc](https://user-images.githubusercontent.com/5279723/39494736-e245c5a2-4d97-11e8-8bda-004f6cc7a26e.png)
![irm_meanwell](https://user-images.githubusercontent.com/5279723/39494740-e8c0d502-4d97-11e8-81f8-b9d41e94246f.png)

Misca1234 commented:
Added the missing 3d shape
Modified the 3d shape to fit the foot print
Added the missing pin 1

![bild](https://user-images.githubusercontent.com/25547797/39590391-0983ddd8-4f01-11e8-9f9f-7ca5643e77ce.png)

Shackmeister commented:
Thanks!
